content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,124 @@ draft: false
 lang: "en"
 ---
 
-TODO
+**Date:** March 26th, 2025
+**Location:** [Virtual/In-Person]  
+**Presenter:** William Hearn (Senior Cloud Architect, GC Cloud One)  
+**Attendees:** ARC Members
+**Objective:** Review the Aurora initiative and secure approval for its **strategic direction and technical architecture**.
+
+---
+
+## 1. Executive Summary
+
+### **Context**
+
+The **Aurora Initiative** is a **unified application hosting platform** designed to standardize **managed Kubernetes cloud offerings** across SSC. It integrates **both proprietary (OpenShift) and OSS (Cloud Native Kubernetes) environments**, drawing from operational experience in the **Cloud Native Platform (CNP)**.
+
+### **Why Now?**
+
+- Current departmental cloud-native strategies are **disjointed, costly, and redundant**.
+- Aurora provides a **governance framework** that reduces inefficiencies, promotes **multi-cloud standardization**, and aligns with SSC’s **GC Cloud One** strategy.
+- The initiative has **received broad buy-in** from multiple departments and aligns with similar government initiatives (BC Gov DevOps, Platform One, etc.).
+
+---
+
+## 2. Architecture Overview
+
+### **2.1 Technical Architecture**
+
+- **Core Services:** Kubernetes (AKS, EKS, GKE, OpenShift)
+- **Networking:** Cilium + BGP (BIRD with route reflectors)
+- **Security Model:** Zero Trust, Gatekeeper (OPA), SCED for on-prem connectivity
+- **CI/CD:** GitLab Runners, Crossplane for cloud resource automation
+- **Observability:** Loki, Prometheus
+- **Multi-Tenancy Approach:** Namespaced isolation, policy enforcement
+
+#### **Key Benefits**
+
+✔️ **Unified API surface** for Kubernetes adoption across SSC.  
+✔️ **Flexible deployment models** (OSS and proprietary).  
+✔️ **Security-first** approach (zero trust, policy-driven enforcement).  
```

Comment:
You can touch on it being an opinionated configuration of the cloud components (such as AKS) and the Aurora components that aligns with GC requirements. Namespace-isolation (Kubernetes resources) is supported by namespace network isolation through network policies, providing a stronger multi-tenant model.

For policy-driven enforcement.. the intention is to empower workload owners to do all activities themselves while being bounded by policy to ensure they cannot violate GC policy.